commands/cli/helptext.go

```diff
@@ -20,7 +20,7 @@ const (
 
 	whitespace = "\r\n\t "
 
-	indentStr = "    "
```

Comment:
No -- this falls under "Terminal formatting / sizing: reduce horizontal indent on help docs".

Let's reduce the tool's horizontal footprint so 80 char terminals don't experience weird line wrap. (Fully dynamic smart wrapping would be even cooler, but that's a bit bigger scope than this.)